test/IDE/complete_func_reference.swift

```diff
@@ -153,25 +151,13 @@ do {
 }
 
 // INT_ANY: Begin completions
-// INT_ANY-DAG: Decl{{.*}}/TypeRelation[Convertible]: intToAny(a:);
```

Comment:
Why did these completions go away? Is it a bug?

Reply:
I asked that same question of @benlangmuir on the other PR. I haven't had a chance to look into it at all, but if it's unexpected we can open a JIRA and look into it.

Reply:
I filed a radar about it already but I thought you determined the change was correct. Until we find out can you leave this test as is for now?

Reply:
@slavapestov I'm not planning on merging anything until @benlangmuir has a chance to reply to my question below.